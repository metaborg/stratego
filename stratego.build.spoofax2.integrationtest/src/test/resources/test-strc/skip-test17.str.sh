#!/bin/sh

java -jar ${STRJ} -i test17.str -la stratego-lib || exit 1 

rm -f test17.str.actual

javac -cp ${STRJ} test17.java || exit 1

java -cp .:${STRJ} -Xmx1024m test17 > test17.str.actual 2>&1

diff test17.str.expected test17.str.actual || (echo "Result not expected; test failed"; exit 1)
