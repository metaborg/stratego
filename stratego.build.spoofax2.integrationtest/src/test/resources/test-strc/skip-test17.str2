module test17

imports libstratego-lib

strategies
  main = rules(Foo: 1 -> 2 with 3); <Foo> 1
