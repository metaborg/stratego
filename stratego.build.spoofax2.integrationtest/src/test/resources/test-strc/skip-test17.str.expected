Internal error: 'with' clause failed unexpectedly in 'Foo', defined in strategy 'main'
	1
	main_0_0
	Foo_0_0
	fetch_elem_1_0
	aux_Foo_0_1
	with_1_1 <==
